plugins/MumblePlugin.h

```diff
@@ -1515,6 +1527,49 @@ struct MUMBLE_API_STRUCT_NAME {
 																			mumble_channelid_t channelID,
 																			const char **description);
 
+#if SELECTED_API_VERSION >= MUMBLE_PLUGIN_VERSION_CHECK(1, 3, 0)
+	/**
+     * Gets the currently used positional data
+     *
+     * @param callerID The ID of the plugin calling this function
+     * @param[out] positionalCoordinates A pointer to the PositionalCoordinates object that should be written to
+     * @returns The error code. If everything went well, STATUS_OK will be returned.
+     *
+     * @since Plugin interface v1.3.0
+	 */
+	mumble_error_t(MUMBLE_PLUGIN_CALLING_CONVENTION *getPositionalData)(mumble_plugin_id_t callerID,
+																		PositionalCoordinates *positionalCoordinates);
+
+	/**
+	 * Gets the currently used positional data context
+	 *
+     * @param callerID The ID of the plugin calling this function
+	 * @param[out] context A pointer to the char * that should be written to,
+	 *   please note that the context has 2 null terminators,
+	 *   the first of which will contain the name of the plugin providing the data
+	 *   whereas the second null terminator contains the actual context data returned by the plugin
+	 * @returns The error code. If everything went well, STATUS_OK will be returned.
+	 *
+	 * @since Plugin interface v1.3.0
+	 */
+	mumble_error_t(MUMBLE_PLUGIN_CALLING_CONVENTION *getPositionalContext)(mumble_plugin_id_t callerID,
+	                                                                            char **context);
```

Comment:
We should probably return a string wrapper and just allocate a buffer inside the API implementation on Mumble's side.
Having to pass in a pre-allocated buffer is not a great API (in particular as there currently is no way of obtaining the minimum size for that buffer)